runtime/tensor.cu

```diff
@@ -12,8 +12,8 @@ struct Tensor {
   };
 
   T* data;
-  nvfuser_index_t size[Dims];
-  nvfuser_index_t stride[AllocDims];
+  Array<nvfuser_index_t, Dims, 1> size;
```

Comment:
Why changed? To use the copy assignment operator?

Reply:
Yes, exactly